app/Console/Kernel.php

```diff
@@ -46,6 +47,8 @@ protected function schedule(Schedule $schedule)
         $schedule->job(new PollForMediaWikiJobsJob)->everyFifteenMinutes();
 
         $schedule->job(new UpdateWikiSiteStatsJob)->dailyAt('19:00');
+
+        $schedule->job(new SendEmptyWikibaseNotificationsJob)->monthly();
```

Comment:
I think we can do this way more often, no? Maybe daily? Probably good to check with product what they expect.

Reply:
Added some question to the ticket: what you are saying @m90 aligns with what I would expect but I believe wasn't the expectation of @dati18 which was that a notification would be repeatedly sent every 30 days. I added https://phabricator.wikimedia.org/T344689#9300833 to try and clarify this discrepancy in understanding

Reply:
ok Evelien has answered. I will fix this now :)